docs/routines/health-watch.md

```
@@ -235,20 +235,28 @@ After all checks, post to `#engineering`:
 POST https://discord.com/api/v10/channels/${DISCORD_ENGINEERING_CHANNEL_ID}/messages
 ```
 
-Message format:
+**All-green run (every check 🟢, no Issues created/updated, no recoveries): post exactly one line** — this is most runs (house style v2, see [`routines/claude/README.md` in `.github`](https://github.com/greenpill-dev-guild/.github/blob/main/routines/claude/README.md#house-style-v2-applies-to-every-posting-routine)):
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)